shared/css/color.scss

```diff
@@ -6,6 +6,7 @@
 
 $black: #000;
 $background_black: #121416;
+$darkest_slate_gray: #25292d;
```

Comment:
Same comment as here: https://github.com/code-dot-org/code-dot-org/pull/44645#discussion_r798114717

Reply:
Will do! probably worth noting that I kind of arbitrarily picked "darkest_slate_gray" as a name since we don't have official brand names for these colors yet.